full/Dockerfile

```diff
@@ -8,6 +8,15 @@ RUN yes | unminimize \
         build-essential \
         htop \
         jq \
+        hub=2.7.0~ds1-1 \
+        zsh=5.5.1-1ubuntu3 \
```

Comment:
We have bash

Reply:
See #76

Reply:
I've used `bash` all my life, but recently while helping out @carl-parrish to get `zsh` working in Gitpod, I realized how nice `zsh` is to use. I can't really explain why, but the experience seemed surprisingly smooth and fast.

Long story short, I don't mind having `zsh` pre-installed in Gitpod (provided that it doesn't make the default image too large), and maybe it can eventually even become the default shell if everyone likes it. (But we'd probably need to have a sort of transition phase where people can easily switch between `bash` and `zsh`, in order not to badly surprise users.)